### DISCUSSION TITLE
Skip tag option in git plugin for some platform pipelines

### DISCUSSION THREAD

**Discussion by timothystone** [semantic-release/semantic-release#2699] [Configuration] [answered]
I'm using the `@semantic-release/git` plugin in a Maven pipeline primarily to generate the changelog. However, the `git` plugin tags the repository, though tagging in the pipeline is accomplished by the Maven Release Plugin.

Maybe I'm missing something, but it doesn't appear that the tagging of the repository can be skipped in either the `changelog` or `git` plugins, especially in pipelines that have facilities to perform the tagging. I've seen some use of the `exec` plugin to "thank" the `git` plugin for the work, but delete the tag in clean up. Seems heavy handed, but a workable solution.

Thoughts on the use of `exec` or is there something I'm missing? Is providing a switch for skipping the tagging a feature the team would consider for a PR? I don't need a fork, just some flexibility for this use case.

**Comment by travi** [comment, marked as answer]
the version tag is added by the core of semantic-release because it is the way that we identify the previous version when determining the commits to evaluate to determine the next version. because it is so core to how semantic-release works, we have no plans to make that part of the process optional
